src/ray/test/run_core_worker_tests.sh

```diff
@@ -40,10 +40,10 @@ fi
 
 REDIS_MODULE="./bazel-bin/libray_redis_module.so"
 LOAD_MODULE_ARGS="--loadmodule ${REDIS_MODULE}"
-STORE_EXEC="./bazel-bin/external/plasma/plasma_store_server"
-RAYLET_EXEC="./bazel-bin/raylet"
-RAYLET_MONITOR_EXEC="./bazel-bin/raylet_monitor"
-MOCK_WORKER_EXEC="./bazel-bin/mock_worker"
+STORE_EXEC="$(bazel info -c dbg $RAY_BAZEL_CONFIG bazel-bin)/external/plasma/plasma_store_server"
+RAYLET_EXEC="$(bazel info -c dbg $RAY_BAZEL_CONFIG bazel-bin)/raylet"
+RAYLET_MONITOR_EXEC="$(bazel info -c dbg $RAY_BAZEL_CONFIG bazel-bin)/raylet_monitor"
+MOCK_WORKER_EXEC="$(bazel info -c dbg $RAY_BAZEL_CONFIG bazel-bin)/mock_worker"
```

Comment:
These are redundant calls, and `bazel` is also a pretty heavy-duty call. Can we factor these prefixes out into a single variable?

Reply:
fixed